docs/observability.md

```diff
@@ -0,0 +1,7 @@
+# Observability
+
+## Logs
+Loki
+
```

Comment:
Some sample queries like `{namespace="myapp"}` for common troubleshooting / monitoring of container logs to provide an initial runbook of useful info an operator may need as they run the cluster / debug apps running on the cluster